include/seqan3/alphabet/nucleotide.hpp

```diff
@@ -44,3 +44,49 @@
 #include <seqan3/alphabet/nucleotide/nucl16.hpp>
 #include <seqan3/alphabet/nucleotide/rna4.hpp>
 #include <seqan3/alphabet/nucleotide/rna5.hpp>
+
+/*!\defgroup nucleotide
+ * \brief Contains the different DNA and RNA alphabet types.
+ * \ingroup alphabet
+ *
+ * \par Concept
+ *
+ * The nucleotide submodule defines seqan3::nucleotide_concept which encompasses all the alphabets defined in the
+ * submodule and refines seqan3::alphabet_concept.
+ *
+ * While it is not required by the concept, all alphabets declared in the module offer an `enum`-like interface
+ * to the individual letters so you can use them as values in assignment or comparison:
```

Comment:
I'd suggest using the plural here "... in assignments or comparisons" :smiley_cat: